proxy/set_proxy.sh

```diff
@@ -1,11 +1,11 @@
-# Non-authenticated HTTP server:
-HTTP_PROXY=http://10.10.1.10:1180
+# Non-authenticated HTTP proxy for HTTPS requests:
+export HTTPS_PROXY=http://10.10.1.10:1180
 
-# Authenticated HTTP server:
-HTTP_PROXY=http://username:password@10.10.1.10:1180
+# Authenticated HTTP proxy for HTTPS requests:
+export HTTPS_PROXY=http://username:password@10.10.1.10:1180
```

Comment:
Nowadays, almost all request is made through HTTPS. So we should put HTTPS before HTTP.

Reply:
Nowadays, almost all requests are made through HTTPS, so we should put HTTPS before HTTP.